graphrag_sdk/ontology.py

```diff
@@ -117,28 +150,28 @@ def add_relation(self, relation: Relation):
         """
         Adds a relation to the ontology.
 
-        Parameters:
+        Args:
             relation (Relation): The relation to be added.
         """
         self.relations.append(relation)
 
-    def to_json(self):
+    def to_json(self, include_all: bool = True) -> dict:
```

Comment:
I think yours will find this confusing, we should reconsider and keep this function as it was.